### PR TITLE
Update package

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Tentacles-Manager/pull/41
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/87

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def build_ext(*args, **kwargs):
 from setuptools import setup, Extension
 from octobot.constants import PROJECT_NAME, VERSION
 
-PACKAGES = find_packages(exclude=["tests"])
```
Comment (Herklos) [inline setup.py:39]: Why not ignoring tests?

Comment (GuillaumeDSM) [reply setup.py:39]: If we ignore tests, the ```python setup.py test``` command can't work, either we remove this command or we let the tests folder.

File: octobot/constants.py [hunk 2]
```diff
@@ -57,9 +58,10 @@
 FORCE_ASYNCIO_DEBUG_OPTION = False
 
 # Files
-CONFIG_FILE = "config.json"
-TEMP_RESTORE_CONFIG_FILE = "temp_config.json"
-CONFIG_FOLDER = "config"
+# Store the path of the octobot directory from this file since it can change depending on the installation path
+# (local sources, python site-packages, ...)
```
Comment (Herklos) [inline octobot/constants.py:62]: :+1:

File: setup.py [hunk 3]
```diff
@@ -79,14 +79,15 @@ def build_ext(*args, **kwargs):
     description='Cryptocurrencies alert / trading bot',
     py_modules=['start'],
     packages=PACKAGES,
-    include_package_data=True,
+    package_data={
+        "": ["config/*", "strategy_optimizer/optimizer_data_files/*"],
```
Comment (Herklos) [inline setup.py:83]: :+1:

**Pull request merged**